src/config.js

```diff
@@ -20,6 +21,21 @@ const evaluationOptions = {
   internalWrites: true,
 };
 
+function getGitCommitHash() {
```

Comment:
I dit it with separate variables. Put json in the environment variable is not the set idea. And it could be a problem when you want to change only one setting and have others in default values.